ngraph/src/ngraph/op/non_max_suppression.cpp

```diff
@@ -300,13 +300,8 @@ void op::v3::NonMaxSuppression::validate_and_infer_types()
                           m_output_type == element::i64 || m_output_type == element::i32,
```

Comment:
nit: output type check in the `validate_inputs` function. should it be just validate or is it taken?